src/commands/status.yml

```diff
@@ -49,7 +54,9 @@ steps:
       shell: /bin/bash
       when: always
       command: |
-        if [ "x" == "x<< parameters.only_for_branch>>" ] || [ "${CIRCLE_BRANCH}" == "<< parameters.only_for_branch>>" ]; then
+        if [[ "x" == "x<< parameters.only_for_branch >>" && "${CIRCLE_BRANCH}" == "<< parameters.only_for_branch >>" ]] ||
+           << parameters.only_for_tag >> && [ -n "${CIRCLE_TAG}" ] ||
```

Comment:
I didn't think bash will evaluate this as true/false, and needs to check for matching the literal string "true" or false is empty..

Reply:
thanks! condition changed. d00f869